pandas/tseries/index.py

```diff
@@ -1597,7 +1597,8 @@ def time(self):
     @property
     def date(self):
         """
-        Returns numpy array of datetime.date. The date part of the Timestamps.
+        Returns numpy array of python datetime.date. The date part of the 
```

Comment:
"array of python datetime.date" -> "array of python datetime.date objects" ? ("objects" is a bit superfluous word, but there should be something plural :-))